src/containers/BooksList.js

```diff
@@ -0,0 +1,20 @@
+import Book from '../components/Book';
+
+const BooksList = () => (
+  <div>
+    <h1>Books List</h1>
+    <table>
+      <tr>
+        <th>Book ID</th>
+        <th>Title</th>
+        <th>Category</th>
+      </tr>
+      <tbody>
+        <Book id={1} title="about memory" category="self improve" />
+        <Book id={2} title="dfdfdfd emory" category="ererererev improve" />
+      </tbody>
+    </table>
+  </div>
+);
+
+export default BooksList;
```

Comment:
- [ ] Kindly connect this component to the Redux store using mapStateToProps to retrieve book objects from the store.
- [ ] Also, please ensure that a book object is passed as a prop to each Book component. So consider mapping through the book array retrieved from the store.